crates/bevy_ecs/src/component.rs

```diff
@@ -251,6 +252,22 @@ impl SparseSetIndex for ComponentId {
     }
 }
 
+/// Prints the shortened type names of the provided [`ComponentId`]s
+///
+/// Uses [`get_short_name`] to strip the module paths of the items, resulting in cleaner lists.
+pub fn display_component_id_types<'a, I: Iterator<Item = &'a ComponentId>>(
+    component_ids: I,
+    components: &Components,
+) -> String {
+    component_ids
+        .map(|id| match components.get_info(*id) {
+            Some(info) => get_short_name(info.name()),
+            None => format!("{:?}", id),
+        })
+        .reduce(|acc, s| format!("{}, {}", acc, s))
+        .unwrap_or_default()
+}
```

Comment:
I'm not sure this function actually _prints_ anything, by looking at the implementation. It just returns a `String`.

Reply:
```suggestion
/// Returns the shortened type names of the provided [`ComponentId`]s
///
/// Uses [`get_short_name`] to strip the module paths of the items, resulting in cleaner lists.
pub fn display_component_id_types<'a, I: Iterator<Item = &'a ComponentId>>(
    component_ids: I,
    components: &Components,
) -> String {
    component_ids
        .map(|id| match components.get_info(*id) {
            Some(info) => get_short_name(info.name()),
            None => format!("{:?}", id),
        })
        .reduce(|acc, s| format!("{}, {}", acc, s))
        .unwrap_or_default()
}
```